src/main/java/com/marklogic/spark/reader/file/RdfSerializer.java

```diff
@@ -2,25 +2,29 @@
 
 import org.apache.jena.graph.Node;
 import org.apache.jena.graph.Triple;
+import org.apache.jena.sparql.core.Quad;
 import org.apache.spark.sql.catalyst.InternalRow;
 import org.apache.spark.sql.catalyst.expressions.GenericInternalRow;
 import org.apache.spark.unsafe.types.UTF8String;
 
 import java.util.Random;
 
 /**
- * Captures the logic from Content Pump for serializing a Jena Triple into a string representation. Note that this does
- * not contain the "escape XML" logic in the MLCP code as we don't care about an XML representation of the triples yet.
- * We just want to return the raw values so they can be added to a Spark row.
+ * Captures the logic from Content Pump for serializing a Jena Triple or Quad into a string representation.
  */
-class TripleSerializer {
+class RdfSerializer {
```

Comment:
I'm not pleased with this name yet. The purpose is "Take a Jena Triple or Quad and produce a Spark row with it". It is "serialization" in that an object is being serialized into some data structure. And I chose "Rdf" since it handles triples and quads.